.github/workflows/integration.yml

```diff
@@ -12,71 +12,34 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# Run secret-dependent integration tests only after /ok-to-test approval
-on:
-  pull_request:
-  repository_dispatch:
-    types: [ok-to-test-command]
-
+# Run secret-dependent integration tests only after approval
 name: Integration tests
-env:
-  IMAGE_NAME: scorecard
-  OUTPUT_PATH: output
+on: pull_request_target
 
 permissions:
   contents: read
   pull-requests: write
 
 jobs:
-  integration-trusted:
+  approve:
```

Comment:
how does this check a reviewer approved? I see only a printed message.

Reply:
It doesn't it only prints a message as you mentioned. The approve happens because of `environment` field.

Reply:
Thanks for clarifying.